java-generic-mqtt/src/main/javadoc/overview.html

```diff
@@ -0,0 +1,85 @@
+<HTML>
+<BODY>
+Cumulocity SDK - Java Generic MQTT Client library provides the classes necessary to interact with the Cumulocity Generic
+MQTT Broker.
+
+<p>The Cumulocity SDK Java Generic MQTT Client library provides means to the following operations:
+<ul>
+    <li>Publishing messages to Cumulocity Generic MQTT Broker via WebSocket protocol</li>
+    <li>Subscribing to messages from Cumulocity Generic MQTT Broker via WebSocket protocol</li>
+</ul>
+<p>
+    Example usage for publishing messages to Generic MQTT Broker via WebSocket:
+
```

Comment:
nitpick: in the two examples we get/provide the websocketUrl differently. In publisher we have a variable, in Subscriber get get it from the properties. Should we do the same in both (I'd vote for getting it from properties)

Reply:
It's changed now.